### PR TITLE
Add Fbits, Tray, Rakuten Digital Commerce, Fastcommerce, MakeShop

### PR DESCRIPTION
**Pull request by nvanthao**
Hi guys,

I have added and tested these new detectors:
- Add Fbits - Ecommerce Platform from Brazil
- Add Tray - Ecommerce Platform from Brazil
- Add Rakuten Digital Commerce - Ecommerce Platform from Brazil
- Add Fastcommerce - Ecommerce Platform from Brazil
- Add MakeShop - Ecommerce Platform from Korea

Cheers,

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -9957,6 +9998,16 @@
       "script": "lucide\\.init(?:\\.min)?\\.js",
       "website": "http://www.translucide.net"
     },
+    "Tray": {
+      "cats": [
+        6
+      ],
+      "icon": "tray.png",
+      "js": {
+        "tray": ""
```
Comment (jvoisin) [inline src/apps.json]: I think that this might be prone to false-positives :/

File: src/apps.json [hunk 2]
```diff
@@ -5724,6 +5744,17 @@
       ],
       "website": "http://mailchimp.com"
     },
+    "MakeShopKorea": {
```
Comment (jvoisin) [inline src/apps.json]: Isn't the name `MakeShop`?

File: src/apps.json [hunk 3]
```diff
@@ -3026,6 +3026,16 @@
       "script": "jquery\\.fancybox(?:\\.pack|\\.min)?\\.js(?:\\?v=([\\d.]+))?$\\;version:\\1",
       "website": "http://fancyapps.com/fancybox"
     },
+    "Fastcommerce": {
+      "cats": [
+        6
+      ],
+      "icon": "Fastcommerce.png",
+      "meta": {
+        "generator": "Fastcommerce"
```
Comment (jvoisin) [inline src/apps.json]: Please use `"generator": "^Fastcommerce$"` if it's possible.

File: src/apps.json (hunk 2 above)
Comment (nvanthao) [reply src/apps.json]: There's MakeShop Japan (https://www.makeshop.jp/) with different pattern compared to Korea (https://www.makeshop.co.kr/)

Cheers,

**Comment by nvanthao** [comment]
Thanks @jvoisin ,

I have updated patterns as per suggestion.

Cheers,

File: src/apps.json (hunk 2 above)
Comment (jvoisin) [reply src/apps.json]: Ok :)

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**